plugins/repoclosure.py

```diff
@@ -102,16 +102,27 @@ def _get_unresolved(self, arch=None):
             to_check = self.base.sack.query().available()
 
         if self.opts.pkglist:
-            to_check = to_check.filter(name=self.opts.pkglist)
+            pkglist_q = self.base.sack.query().filter(empty=True)
+            errors = []
+            for pkg in self.opts.pkglist:
+                pkg_q = to_check.filter(name=pkg)
```

Comment:
According to doc:
``--pkg <pkg``
    Check closure for this package only.

pkg could be not only pkg name. What about to use a subject with_nevra only?